pkg/backup/backup/backup_manager.go

```diff
@@ -723,6 +723,11 @@ func (bm *backupManager) makeBRBackupJob(backup *v1alpha1.Backup) (*batchv1.Job,
 		}
 		args = append(args, fmt.Sprintf("--mode=%s", v1alpha1.BackupModeVolumeSnapshot))
 	default:
+		klog.Infof("Backup cluster manifests for snapshot backup %s/%s", ns, name)
```

Comment:
please add a switch (by default false) to control if manifest backup is needed for snapshot backup since this change add more items to backup bucket.